datasets/snow_simplified_japanese_corpus/snow_simplified_japanese_corpus.py

```diff
@@ -0,0 +1,168 @@
+    title = "Simplified Corpus with Core Vocabulary",
+    author = "Maruyama, Takumi  and
+      Yamamoto, Kazuhide",
+    booktitle = "Proceedings of the Eleventh International Conference on Language Resources and Evaluation ({LREC} 2018)",
+    month = may,
+    year = "2018",
+    address = "Miyazaki, Japan",
+    publisher = "European Language Resources Association (ELRA)",
+    url = "https://www.aclweb.org/anthology/L18-1185",
+}
+
+@inproceedings{yamamoto-2017-simplified-japanese,
+    title = "やさしい⽇本語対訳コーパスの構築",
+    author = "⼭本 和英  and
+      丸⼭ 拓海  and
+      ⾓張 ⻯晴  and
+      稲岡 夢⼈  and
+      ⼩川 耀⼀朗  and
+      勝⽥ 哲弘  and
+      髙橋 寛治",
+    booktitle = "言語処理学会第23回年次大会",
+    month = 3月,
+    year = "2017",
+    address = "茨城, 日本",
+    publisher = "言語処理学会",
+    url = "https://www.anlp.jp/proceedings/annual_meeting/2017/pdf_dir/B5-1.pdf",
+}
+
+@inproceedings{katsuta-yamamoto-2018-crowdsourced,
+    title = "Crowdsourced Corpus of Sentence Simplification with Core Vocabulary",
+    author = "Katsuta, Akihiro  and
+      Yamamoto, Kazuhide",
+    booktitle = "Proceedings of the Eleventh International Conference on Language Resources and Evaluation ({LREC} 2018)",
+    month = may,
+    year = "2018",
+    address = "Miyazaki, Japan",
+    publisher = "European Language Resources Association (ELRA)",
+    url = "https://www.aclweb.org/anthology/L18-1072",
+}
+"""
+
+_DESCRIPTION = """\
+About SNOW T15: \
+The simplified corpus for the Japanese language. The corpus has 50,000 manually simplified and aligned sentences. \
+This corpus contains the original sentences, simplified sentences and English translation of the original sentences. \
+It can be used for automatic text simplification as well as translating simple Japanese into English and vice-versa. \
+The core vocabulary is restricted to 2,000 words where it is selected by accounting for several factors such as meaning preservation, variation, simplicity and the UniDic word segmentation criterion.
+For details, refer to the explanation page of Japanese simplification (http://www.jnlp.org/research/Japanese_simplification). \
+The original texts are from "small_parallel_enja: 50k En/Ja Parallel Corpus for Testing SMT Methods", which is a bilingual corpus for machine translation. \
+\
+About SNOW T23: \
+An expansion corpus of 35,000 sentences rewritten in easy Japanese (simple Japanese vocabulary) based on SNOW T15. \
+The original texts are from "Tanaka Corpus" (http://www.edrdg.org/wiki/index.php/Tanaka_Corpus).
+"""
+
+_HOMEPAGE = "http://www.jnlp.org/SNOW/T15, http://www.jnlp.org/SNOW/T23"
+
+_LICENSE = "CC BY 4.0"
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+_URLs = {
+    "snow_t15": "https://filedn.com/lit4DCIlHwxfS1gj9zcYuDJ/SNOW/T15-2020.1.7.xlsx",
+    "snow_t23": "https://filedn.com/lit4DCIlHwxfS1gj9zcYuDJ/SNOW/T23-2020.1.7.xlsx",
+}
+
+
+class SnowJapanese(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class SnowSimplifiedJapaneseCorpus(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `snow_simplified_japanese_corpus`